tower_cli/models/base.py

```diff
@@ -1215,6 +1215,22 @@ def create(self, **kwargs):
         return self.write(create_on_missing=True, **kwargs)
 
     @resources.command(ignore_defaults=True)
+    def copy(self, pk=None, **kwargs):
+        """Copy an object.
+        """
+        orig = self.read(pk, fail_on_no_results=True,
+                         fail_on_multiple_results=True)
+        orig = orig['results'][0]
+        # Remove default values (anything where the value is None).
+        self._pop_none(kwargs)
+
+        newresource = copy(orig)
+        newresource.update(kwargs)
+        newresource['id'] = None
+        newresource['name'] += " @ %s" % (time.strftime('%X'))
```

Comment:
yeah, good catch.

any reason you used explicit `time.strftime('%I:%M:%S %p')` and not `%X` which should (at least sometimes) match the user locale?